aws-ts-static-website/src/lambda-language-redirect.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+exports.handler = (event, context, callback) => {
+    const request = event.Records[0].cf.request;
+    const languageHeader = request.headers['accept-language'];
+
+    // shortcut invalid requests
+    if (request.uri != '/' || typeof languageHeader === 'undefined') {
+        callback(null, request);
+        return;
+    }
+
+    const clientLanguages = languageHeader[0].value;
+    console.log('accept-language header from client:', clientLanguages);
```

Comment:
I still haven’t found out where these logs go! The [documentation](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-testing-debugging.html) says that this will be logged through Cloudwatch and in the nearest region, but that didn’t work for me. Would the logging require additional AWS infrastructure in the Pūlumi program?

Reply:
An extra policy is required to allow the lambda writing the logs, see my latest commit